JenkinsfileRT

```diff
@@ -87,7 +87,7 @@ bc0.conda_packages = [
 bc0.pip_reqs_files = ['requirements-sdp.txt']
 bc0.build_cmds = [
     "pip install -e .[test]",
-    "pip install pytest-xdist pytest-sugar ddtrace",
+    "pip install pytest-xdist ddtrace",
```

Comment:
These Jenkins workflow changes disable fancy symbols and coloring of the pytest output on Jenkins. These normally look great (and are useful when running locally) in a terminal because its inserting escaped unicode characters and color codes. However, it pollutes log output stored by Jenkins because all those escape sequences show up in that. Thus explicitly turning them off will make that log much easier to  read.